src/config/links.js

```diff
@@ -13,6 +13,11 @@ export const links = {
   email_request_ccdl: 'requests@ccdatalab.org',
   // refine.bio
   refinebio: 'https://www.refine.bio',
+  refinebio_data_request: {
+    experiment: 'https://www.refine.bio/experiments/',
+    search: 'https://www.refine.bio/search?search='
+  },
+  refinebio_email_logo: 'https://s3.amazonaws.com/refinebio-email/logo-2x.png',
```

Comment:
Move these outside of `config.links`, I think `requests` would make more sense. Also, we can just host this image in the public folder of this application instead of creating an external dependency.

Reply:
Thank you for this input! I've made the changes at ea59aa1f728e6f120e855a60fe45fd11e20cb8cf.

Reply:
The `req` (request) object contains the read only property `url`. I think instead of explicitly setting this we could just do something like:

```javascript
host = new URL(req.url).hostname
```

Docs:
- https://nextjs.org/docs/app/api-reference/functions/next-request
- https://developer.mozilla.org/en-US/docs/Web/API/Request (NextRequest inherits from this).

Reply:
Thank you for this great suggestion! I've applied it at 72250fd05dc683f161565c21c00ec069cf7ea9c7 👍